heroku/cli
# refactor(spaces): Move command spaces:peering:destroy to oclif

### Description
[GUS work item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b1ax6YAA/view)

This converts `spaces:peerings:destroy` to oclif and its associated tests. This also moves the `spaces:peering:info` to `spaces:peerings:info` alongside other `peerings` commands. Not sure why the `info` command lacked the `s`, but probably was an oversight.

### Testing
Please use a test space with these instructions. This will make (possibly breaking) changes to a space.

1) Pull down this branch and run `yarn build`
2) Run `./bin/run spaces:peerings:destroy --space SPACE_NAME PCXID` against an existing test space. 
3) See that the peering has been destroyed by running `heroku spaces:peering:info SPACE_NAME`
4) Compare with prod
